components/authz-service/storage/postgres/migration/legacy/iam_migration_test.go

```diff
@@ -382,3 +396,102 @@ func memberSliceToStringSlice(m []v2Member) []string {
 	}
 	return memberSlice
 }
+
+// v1LegacyPolicies is a consolidated list of the v1LegacyPolicies
+func v1LegacyPolicies() []v2Policy {
```

Comment:
Where do these come from? Wondering what kind of maintenance burden this is adding. At a minimum, should add a comment saying "these statements should be kept in sync with _xyz_..."

Reply:
@msorens these are the policies from v1 that are ported in the `migratev1policies` call, and they (like everything under `legacy`) should remain untouched after `force-upgrade` is merged. I will add a comment to that effect.